financial-connections/src/main/java/com/stripe/android/financialconnections/ui/FinancialConnectionsSheetNativeActivity.kt

```diff
@@ -12,23 +14,49 @@ import androidx.navigation.NavHostController
 import androidx.navigation.compose.NavHost
 import androidx.navigation.compose.composable
 import androidx.navigation.compose.rememberNavController
+import com.airbnb.mvrx.MavericksView
 import com.airbnb.mvrx.viewModel
+import com.airbnb.mvrx.withState
 import com.stripe.android.financialconnections.navigation.NavigationDirections
+import com.stripe.android.financialconnections.presentation.CreateBrowserIntentForUrl
+import com.stripe.android.financialconnections.presentation.FinancialConnectionsSheetNativeViewEffect.OpenAuthFlowWithUrl
 import com.stripe.android.financialconnections.presentation.FinancialConnectionsSheetNativeViewModel
 
-internal class FinancialConnectionsSheetNativeActivity : AppCompatActivity() {
+internal class FinancialConnectionsSheetNativeActivity : AppCompatActivity(), MavericksView {
 
     val viewModel: FinancialConnectionsSheetNativeViewModel by viewModel()
 
+    private val startBrowserForResult = registerForActivityResult(StartActivityForResult()) {}
```

Comment:
is this intentionally left blank?

Reply:
good catch, we don't actually need to start the browser for result. https://github.com/stripe/stripe-android/pull/5140/commits/2e81e24fad068424f813a6d96344686facc5574f

- if flow completed, browser deep-links to `FinancialConnectionsSheetRedirectActivity` (as with the full-web flow)
- if flow closed, we go back to where we left off in the native side, the bank picker.